Milestone 1: Project-Setup

smcommits opened the pull request:
This is a pull request for milestone 1 of the project Tic Tac Toe.
As a part of milestone 1, we have: 

- Created a feature branch named project-setup
- Added lib folder using .gitkeep
- Added main.rb to the bin folder

smcommits committed:
Create feature branch, add lib and bin folder, add main.rb

Created feature branch named project-setup
added lib folder using .gitkeep
added main.rb to the bin folder

mahmoud717 reviewed (changes requested):
### Changes required ♻️ 
**Hi @smcommits & @Lameck1**
nice job with the project so far, but there are still some adjustments to be made:
---
- [ ]  please make sure that the main file is executable, you can find more info on that [here](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/)

- [ ] make sure to create a lib folder with a .gitkeep file inside of it
---
if you need any further clarifications please contact me on slack `Mahmoud Mohammad`

Commit:
Add /bin/.gitkeep, Make main.rb executable

Commit:
Add /lib/.gitkeep, Make main.rb executable

Added .gitkeep file to lib directory
Made main.rb file executable by changing permission

Commit:
Add /lib/.gitkeep, Make main.rb executable

Added .gitkeep file to lib directory
Made main.rb file executable by changing permission

Commit:
Merge branch 'feature/project-setup' of https://github.com/Lameck1/Tic-Tac-Toe into feature/project-setup

umairarshadbutt reviewed (approved):
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @smcommits  and @Lameck1! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)
